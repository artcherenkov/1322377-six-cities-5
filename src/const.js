export const HousingType = {
  APARTMENT: `apartment`,
  ROOM: `room`,
  HOUSE: `house`,
  HOTEL: `hotel`
};

export const Cities = {
  PARIS: `PARIS`,
  COLOGNE: `COLOGNE`,
  BRUSSELS: `BRUSSELS`,
  AMSTERDAM: `AMSTERDAM`,
  HAMBURG: `HAMBURG`,
  DUSSELDORF: `DUSSELDORF`
};

export const MapType = {
  PROPERTY: `property__map`,
  CITIES: `cities__map`
};

export const OffersListType = {
  NEAR: `near-places__list`,
  CITIES: `cities__places-list tabs__content`
};

export const SortType = {
  DEFAULT: `DEFAULT`,
  POPULAR: `Popular`,
  PRICE_LOW_TO_HIGH: `Price: low to high`,
  PRICE_HIGH_TO_LOW: `Price: high to low`,
  TOP_RATED: `Top rated first`
};
