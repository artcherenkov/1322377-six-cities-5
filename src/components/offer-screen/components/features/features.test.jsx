import React from 'react';
import renderer from 'react-test-renderer';

import Features from './features';
import {HousingType} from "../../../../const";

it(`Features is rendered correctly`, () => {
  const tree = renderer.create(
      <Features
        type={HousingType.APARTMENT}
        bedrooms={4}
        maxAdults={3}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
