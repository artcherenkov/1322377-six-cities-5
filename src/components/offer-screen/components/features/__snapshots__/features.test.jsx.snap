// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features is rendered correctly 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    Apartment
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    4 Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    3 Adults
  </li>
</ul>
`;
